recipes/quast/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "quast" %}
-{% set version = "4.6.0" %}
-{% set hash = "6ad4ef33b5f11e1d2efd3e15a285d0c8" %}
+{% set version = "4.6.1" %}
+{% set hash = "29118bff957af6d1b36136735450ddc8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 1
+  number: 2 
```

Comment:
This should start from 0 if you increase the version number.

Reply:
Maybe I understand build-number policy incorrectly. Initially I set it to 0 when I made the first commit changing Quast version from 4.5 to 4.6.1. After that, I got failed tests and made several changes in the recipe only (not in Quast v.4.6.1). I thought that I need to change build-number after such changes (I did this twice in this patch, so build number became 2).

Reply:
You nearly got it right, with the exception that the build numbers are only "manifested" for builds that are released to the Conda channel, i.e., (usually) builds that happened on the `master` branch. That means that for additional changes you make before your PR is merged, you won't have to increase the number as no builds were actually uploaded. Hence `quast 4.6.1` is always at build number `0` for this PR.

Reply:
Ok, got it, thanks!